nvflare/job_config/fed_job.py

```diff
@@ -218,6 +276,32 @@ def to(
         if self._components:
             self._add_referenced_components(obj, target)
 
+    def to_server(
+        self,
+        obj: Any,
+        tasks: List[str] = None,
+        gpu: Union[int, List[int]] = None,
+        filter_type: FilterType = None,
+        id=None,
+    ):
+        if isinstance(obj, Executor):
+            raise ValueError("Use `job.to(executor, <client_name>)` or `job.to_clients(executor)` for Executors.")
+
+        self.to(obj=obj, target=SERVER_SITE_NAME, tasks=tasks, gpu=gpu, filter_type=filter_type, id=id)
+
+    def to_clients(
+        self,
+        obj: Any,
+        tasks: List[str] = None,
+        gpu: Union[int, List[int]] = None,
+        filter_type: FilterType = None,
+        id=None,
+    ):
+        if isinstance(obj, Controller):
+            raise ValueError('Use `job.to(controller, "server")` or `job.to_server(controller)` for Controllers.')
+
+        self.to(obj=obj, target=ALL_SITES, tasks=tasks, gpu=gpu, filter_type=filter_type, id=id)
```

Comment:
So "to_clients" will always deploy to ALL_SITES, users can't choose which one "to" deploy?

We should add this in the docstring explanation

Reply:
yes, using jus to(..), the user can specify the target.